CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 3.2.7 (TBD)
+
+### Bug fixes
+
+* Gradle 4.8-rc-1 compat fix - use def when finding the buildType
```

Comment:
This summary seems unclear. It should say is happening from a user perspective, e.g. "Add compatibility with Gradle 4.8-rc1"